rpc/evm/src/evm_api.rs

```diff
@@ -0,0 +1,20 @@
+//! EVM rpc interface.
+
+use crate::{bytes::Bytes, call_request::CallRequest};
+use ethereum_types::U256;
+use jsonrpc_core::Result;
+use jsonrpc_derive::rpc;
+
+pub use rpc_impl_EVMApi::gen_server::EVMApi as EVMApiServer;
+
+/// EVM rpc interface.
+#[rpc(server)]
+pub trait EVMApi<BlockHash> {
+	/// Call contract, returning the output data.
+	#[rpc(name = "eth_call")]
```

Comment:
```suggestion
	#[rpc(name = "evm_call")]
```

merge both into one and change return type to `Result<(Bytes, u64)>`